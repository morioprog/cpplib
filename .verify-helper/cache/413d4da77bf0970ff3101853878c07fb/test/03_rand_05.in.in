155 223
56 23
56 151
56 119
146 33
14 86
14 150
3 87
3 115
141 37
141 111
141 20
28 81
28 138
131 15
131 147
53 84
50 7
50 102
50 135
37 130
37 74
11 31
55 87
16 115
95 36
95 29
48 87
6 43
6 70
128 104
128 100
25 122
25 87
25 45
129 97
129 20
98 7
98 91
130 81
130 34
130 102
97 88
97 65
97 8
60 17
110 77
110 135
110 72
101 94
101 119
101 65
15 147
1 7
1 147
1 2
118 5
118 67
133 151
133 92
133 104
74 132
74 24
74 93
81 102
81 150
68 49
12 139
107 61
107 99
36 32
36 51
63 70
89 4
124 134
124 27
41 132
41 112
7 134
7 71
122 119
122 103
61 8
62 80
151 19
152 4
152 20
106 117
114 79
114 99
84 21
84 27
84 105
145 69
145 9
35 65
85 9
85 117
148 126
148 135
111 117
111 149
75 119
75 26
75 0
147 134
147 0
127 91
127 149
79 80
79 54
79 19
121 20
88 103
88 40
31 123
31 2
31 91
86 103
33 90
67 24
67 21
67 135
80 66
80 83
126 34
126 135
126 0
154 120
92 90
92 38
92 45
87 139
34 90
34 142
34 19
49 142
49 9
96 136
13 54
13 150
123 65
10 54
10 27
65 71
65 19
90 21
90 93
138 142
22 109
22 78
22 100
69 70
69 113
70 54
70 52
17 4
17 20
108 113
108 116
108 136
76 115
76 27
76 150
104 115
104 78
104 20
66 0
113 2
113 117
64 21
64 2
64 77
115 21
115 102
115 20
24 54
142 26
142 135
142 45
21 134
120 54
120 105
9 117
40 26
40 71
40 93
32 91
38 27
4 52
4 135
54 136
54 93
100 26
100 139
77 105
77 93
77 20
83 52
83 102
83 51
136 135
136 149
136 51
52 102
52 117
27 8
139 19
91 105
91 150
91 72
102 19
102 150
45 51
117 149
117 150
117 20
8 72
8 20
149 20
72 51
72 0
51 0
0 20
