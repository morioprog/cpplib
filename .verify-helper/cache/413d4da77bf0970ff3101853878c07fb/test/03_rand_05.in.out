154
153
152
148
146
145
144
143
141
140
137
133
131
129
128
127
126
125
124
121
120
118
114
111
110
108
116
107
106
104
101
99
98
97
96
95
94
92
89
88
85
82
79
76
75
73
69
113
68
67
64
77
63
62
80
83
66
61
60
59
58
57
56
151
55
53
84
50
49
48
47
46
44
42
41
112
40
39
38
37
74
132
130
36
35
34
33
90
32
30
29
28
138
142
81
25
45
122
119
24
23
22
100
26
78
109
18
17
16
15
14
86
103
13
12
11
31
91
105
123
65
10
27
9
8
72
6
70
54
93
136
51
43
5
4
135
52
117
149
3
115
102
150
21
87
139
19
1
2
147
7
71
134
0
20
